xml/deployment_yast_installer.xml

```diff
@@ -2440,13 +2440,13 @@ sle-live-patching 8c541494</screen>
     </para>
    </important>
    <para>
-    This category allows hardening your system with &openscap; security
+    This category allows hardening of your system with &openscap; security
```

Comment:
I disagree on this one. Hardening is a verb not a noun here. You harden your system, this is a process.

Reply:
I've reverted it, thanks